client/core/trade.go

```diff
@@ -159,13 +159,15 @@ type trackedTrade struct {
 	options            map[string]string
 	redemptionReserves uint64
 	redemptionLocked   uint64
+	refundReserves     uint64
+	refundLocked       uint64
 }
 
 // newTrackedTrade is a constructor for a trackedTrade.
 func newTrackedTrade(dbOrder *db.MetaOrder, preImg order.Preimage, dc *dexConnection, epochLen uint64,
 	lockTimeTaker, lockTimeMaker time.Duration, db db.DB, latencyQ *wait.TickerQueue, wallets *walletSet,
 	coins asset.Coins, notify func(Notification), formatDetails func(Topic, ...interface{}) (string, string),
-	options map[string]string, redemptionReserves uint64) *trackedTrade {
+	options map[string]string, redemptionReserves uint64, refundReserves uint64) *trackedTrade {
```

Comment:
Unimportant, but the preferred syntax would be `redemptionReserves, refundReserves uint64` without the redundant type.